### PR TITLE
Minor changes FHIR Mapper

### PR DESCRIPTION
**Pull request by alexzautke**

### CODE REVIEW

File: mappingengine/start.rst [hunk 1]
```diff
@@ -3,16 +3,24 @@
 ===============
 Getting Started
 ===============
-The FHIR Mapper (a .Net FHIR Mapping Engine) is available as a paid addon to the Vonk FHIR server. `Get in touch <https://fire.ly/contact/>`_ with us if you're interested in obtaining it for your needs.
+
+The FHIR Mapper is available as a paid plugin to the Vonk FHIR server. `Get in touch <https://fire.ly/contact/>`_ with us if you're interested in obtaining it for your needs. ::
+
+
+    Documentation is currently under construction! Please be patient.
+
```
Comment (vadi2) [inline mappingengine/start.rst]: Can you revert this block?

File: mappingengine/start.rst [hunk 2]
```diff
@@ -3,16 +3,24 @@
...
 
 Running transformations
 -----------------------
 
-Once you have setup the .Net FHIR Mapping Engine, you can start :ref:`setup_transforms`.
\ No newline at end of file
+
+Once you have setup the FHIR Mapper, you can start :ref:`setup_transforms`.
+
+Alternatively, you can try the FHIR Mapper in our public test sandbox: https://vonk.fire.ly/R3. The rest of the documentation will be using the public sandbox as a point of reference.
```
Comment (vadi2) [inline mappingengine/start.rst]: Same here. I've edited this content while you changed the mapping engine to plugin.

File: mappingengine/transformsetup/mappingfile.rst [hunk 3]
```diff
@@ -3,9 +3,15 @@
...
+- `FHIR Mapping Language syntax <https://www.hl7.org/fhir/mapping-language.html>`_ (Human-readable)
+- `FHIR StructureMap resource <https://www.hl7.org/fhir/stu3/structuremap.html>`_ (Machine-readable)
 
-1. ``POST http(s)://<vonk-endpoint>/$convert`` with the body as your mapping file and the ``Content-Type`` header set to ``text/fhir-mapping;charset=utf-8`` to convert your mapping file to a StructureMap.
+These two formats are isomorphic and can be converted between each other. 
+
+The FHIR Mapper operates on a StructureMap resource, so let's convert the mapping file to a `StructureMap <https://www.hl7.org/fhir/structuremap.html>`_ resource and upload to our Vonk. 
+
+1. ``POST http(s)://<vonk-endpoint>/$convert`` with the body as your mapping file and the ``Content-Type`` header set to ``text/fhir-mapping;charset=utf-8`` to convert your mapping file to a StructureMap. The operation parses your mapping file to check it is valid. You will receive an OperationOutcome if an syntax error is encountered inclduing a hint on how to fix it.
```
Comment (vadi2) [inline mappingengine/transformsetup/mappingfile.rst]:
```suggestion
1. ``POST http(s)://<vonk-endpoint>/$convert`` with the body as your mapping file and the ``Content-Type`` header set to ``text/fhir-mapping;charset=utf-8`` to convert your mapping file to a StructureMap. The operation parses your mapping file to check it is valid. You will receive an OperationOutcome if an syntax error is encountered, including a hint on how to fix it.
```

File: mappingengine/enginesetup/configuration.rst [hunk 4]
```diff
@@ -3,9 +3,9 @@
 Configuring FHIR Mapper on Vonk
 ======================================
 
-To configure the mapping engine in your Vonk installation, follow the steps below. If you don't have a copy yet, `get in touch <https://fire.ly/contact/>`_ with us if you're interested in purchasing it for your needs.
+To configure the FHIR Mapper in your Vonk installation, follow the steps below. If you don't have a copy yet, `get in touch <https://fire.ly/contact/>`_ with us if you're interested in purchasing it for your needs.
 
-Load the engine
+Load the plugin
```
Comment (vadi2) [inline mappingengine/enginesetup/configuration.rst]: Why prefer to call it a simple plugin as opposed to a more powerful engine?

File: mappingengine/start.rst (hunk 2 above)
Comment (alexzautke) [reply mappingengine/start.rst]: What do I need to change here? Why did you delete the part of the public endpoint?

Comment (vadi2) [reply mappingengine/start.rst]: Because I checked the public endpoint and the plugin wasn't there, not in `/metadata` at least :O

Comment (alexzautke) [reply mappingengine/start.rst]: Not yet, will be next week

Comment (vadi2) [reply mappingengine/start.rst]: Let's put it in there when it's live, better than having a mistake

**Pull request merged**